source/settings/show-kerberos-password.txt

```diff
@@ -0,0 +1,110 @@
+.. _compass-kerberos-password:
+
+===================================
+Display the Kerberos Password Field
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+If you use :manual:`Kerberos authentication </core/kerberos/>` to connect to 
+your MongoDB deployment, you can enable the ``showKerberosPasswordField`` 
+option on |compass|. 
+
+About This Task 
+---------------
+
+By default, |compass-short| disables the ``showKerberosPassowrdField`` option.
+
+If you enable the ``showKerberosPasswordField``, |compass| displays the 
+:guilabel:`Provide password directly` option, which contains the 
+:guilabel:`Password` field. You can find the Kerberos :guilabel:`Password` 
+under the :guilabel:`Advanced Connection Options > Authentication` tab on the 
+|compass-short| connection form. 
```

Comment:
I think it’s worth mentioning that this option is disabled by default because typically, using the password field is just not necessary or helpful for Kerberos.

Reply:
That was mentioned on line 22 👍

Reply:
Line 22 only mentions that it is disabled, though

Reply:
Ah sorry I think I misunderstood. Did you mean also specifying the reason for why it's disabled by default?